src/legacy/ui/public/vis/editors/default/default_editor.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useEffect, useRef, useState, useCallback } from 'react';
+
+import { start as embeddables } from '../../../../../core_plugins/embeddable_api/public/np_ready/public/legacy';
+import { EditorRenderProps } from '../../../../../core_plugins/kibana/public/visualize/np_ready/types';
+import { VisualizeEmbeddable } from '../../../../../core_plugins/kibana/public/visualize_embeddable';
+import { VisualizeEmbeddableFactory } from '../../../../../core_plugins/kibana/public/visualize_embeddable/visualize_embeddable_factory';
+import {
```

Comment:
I'm also working on eui component here: https://github.com/elastic/eui/pull/2701
So it will bee replaced with this when it ready. I'm not sure if I need to move it to `kibana_react` for such an intermediate layer?

Reply:
Ah right, let's leave it there for now and remove it in both places once the EUI component is ready.